services/report/languages/pycoverage.py

```diff
@@ -22,23 +22,54 @@ def matches_content(self, content, first_line, name) -> bool:
             and "files" in content
         )
 
-    def _convert_testname_to_label(self, testname, labels_table):
+    def _convert_testname_to_label(self, testname) -> str:
```

Comment:
I find the `testname` terminology a little confusing since I wasn't sure what the difference between a label and a test name is.  Maybe `_normalize_label` would describe this behavior?